lib/modules/easterEgg.js

```diff
@@ -23,11 +21,10 @@ module.go = () => {
 			notificationID: 'konami',
 		});
 		setTimeout(() => $baconBit.addClass('makeitrain'), 500);
-	};
+	});
 	konami.iphone.load = () => {}; // nix touch support; occasional false positives on touchscreen laptops
-	konami.load();
 };
 
 export function konamiActive() {
-	return konami && konami.almostThere;
+	return konami;
```

Comment:
Ah, ok that makes sense. I'll take a stab at solving this problem. Do you have a use case that I can test with? I'm not sure what other actions are being interrupted by the konami code.